core/src/main/java/io/dekorate/project/BuildInfo.java

```diff
@@ -28,41 +28,47 @@
   private final String version;
   private final String packaging;
   private final String buildTool;
+  private final String buildToolVersion;
   private final Path outputFile;
   private final Path classOutputDir;
   private final Path resourceDir;
 
   public BuildInfo() {
-    this(null, null, DEFAULT_PACKAGING, DEFAULT_BUILD_TOOL, null, null, null);
+    this(null, null, DEFAULT_PACKAGING, DEFAULT_BUILD_TOOL, null, null, null, null);
   }
 
   /**
    * Constructor
    * @param name                  The project name (e.g. maven artifactId).
    * @param version               The project version (e.g. maven version).
    * @param packaging             The project packaging (e.g. jar, war).
+   * @param buildTool             The project build tool (e.g. maven, gralde, sbt)
+   * @param buildToolVersion      The build tool version
    * @param outputFile            The output file (the path to the actual jar, war etc).
    * @param classOutputDir        The resource output directory (e.g. target/classes, build/classes/main/java etc).
    */
-  public BuildInfo(String name, String version, String packaging, String buildTool, Path outputFile, Path classOutputDir) {
-    this(name, version, packaging, buildTool, outputFile, classOutputDir, classOutputDir);
+  public BuildInfo(String name, String version, String packaging, String buildTool, String buildToolVersion, Path outputFile, Path classOutputDir) {
```

Comment:
What happens if we don't pass the build tool version? Asking because we instantiate `BuildInfo` in Quarkus and we don't know the build tool version there. Is it going to continue to work? I assume the answer is yes because from what I can tell this version is only needed for Tekton

Reply:
It doesn't really matter what quarkus does, we do get the version from `mvn version` anyway.